### DISCUSSION TITLE
Remove underline on navigation bar when the text is hover

### DISCUSSION THREAD

**Discussion by thomasbnt** [forem/forem#15869] [Feature Requests]
Hello all ! 

Since the latest update of the design on the **navigation bar**, I have a lot of trouble with the way it looks. We already have a background that is getting darker and that shows the link is over. But why did you add this underline?

![Navigation bar with underline when you hover the text](https://user-images.githubusercontent.com/14293805/147188376-108a05b3-456f-45ac-889f-3546620fea7a.png)

Maybe remove that ?

**Comment by aitchiss** [comment]
Hi @thomasbnt! A big part of why this underline appeared is that we are working to improve our design language around buttons vs links in the app, largely for accessibility purposes. I'm planning to write up a DEV post soon with some more detail about the accessibility motivations around these changes, but if you're interested to see the broader design direction around this, feel free to check out [our Storybook](https://storybook.forem.com/), particularly the 'BETA' section at the bottom with the new Links, Buttons and CTAs (these will very shortly be moving out of the BETA and into the main Components section).

**Reply by AlexaHills** [reply]
Thanks for your information. It will be very useful for all.

**Comment by Cyrus7462** [comment]
According to my experience you have applied double properties and characters to the navigation. As we were facing the same issue on our [lottery sambad old](https://www.winscenes.in/lottery-sambad-old-results/) website. But when I contacted to the developer so he told me that you have applied double properties on the nav. As I provided the access to him and he fixed it in a few minutes.

**Comment by henkjankrijger** [comment]
In my case Section looking green, but when i put mouse pointer on it it became looking like an a href, but i want it stay just the same without underline and changing color. more color visit to my website [how to delete instagram](https://yoo.rs/how-to-delete-instagram-account-1643779930.html?Ysid=1)

Is it possible to achieve without changing css or with minimum css cahnge?

**Comment by Jaimeanderson** [comment]
To remove the underline on a navigation bar when the text is being hovered, you can use CSS. Here is an example code snippet that you can modify to suit your specific website's navigation bar:

css
Copy code
/* Select the navigation links */
.navbar a {
  /* Set text-decoration to none to remove underline by default */
  text-decoration: none;
}

/* Style the navigation links when hovered */
.navbar a:hover {
  /* Set text-decoration to none to remove underline when hovered */
  text-decoration: none;
}
In the above code, .navbar a selects the navigation links and sets the text-decoration property to none, which removes the underline by default. .navbar a:hover applies styles to the links when they are hovered, and again sets text-decoration to none, which removes the underline when the links are being hovered.

**Reply by thomasbnt** [reply]
Arrg spam.

**Comment by CamiloCamilo12345678** [comment]
To remove the underline on a navigation bar when the text is hovered, you can use CSS. Here is an example:

css
Copy code
/* Remove underline on navigation links */
nav a {
  text-decoration: none;
}

/* Add underline on hover */
nav a:hover {
  text-decoration: underline;
}
In this code, we first set the text-decoration property to none for all the links in the navigation bar. This will remove any underline that may be present on the links.

Then, we add a CSS rule for the :hover

**Reply by thomasbnt** [reply]
Arrg spam.

**Comment by CamiloCamilo12345678** [comment]
Hello I am planning to write up a DEV post soon with some more detail about the accessibility motivations around these changes,

**Comment by CamiloCamilo12345678** [comment]
To remove the underline on a navigation bar when the text is hovered over, you can use CSS. Here's an example:

css
Copy code
/* Remove underline on hover */
nav a:hover {
  text-decoration: none;
}
This code targets the <a> elements within your navigation bar (assuming your navigation bar is wrapped in a <nav> element) and removes the text-decoration property when the text is hovered over. The text-decoration property controls the underline, so setting it to none removes the underline.

You can add this CSS to your stylesheet or add it directly to the <head> section of your HTML document using a <style> tag.

**Comment by thomasbnt** [comment]
I close this discussion, **but not closed** due to spams.

**Comment by axrmyinluv21** [comment]
Thanks for your information. It will be very useful for all.

**Comment by CamiloCamilo12345678** [comment]
To remove the underline on a navigation bar when the text is being hovered, you can use CSS code. I was also facing this issue in my website [Pembio ](https://pembio.com/) menu and my problem was solved you should also follow these steps Here's an example:

css
Copy code
/* Remove underline on navigation bar */
nav a {
  text-decoration: none;
}

/* Add underline on hover */
nav a:hover {
  text-decoration: underline;
}
In the above code, the first block sets the default style of the navigation bar links to have no text decoration (i.e., no underline). The second block adds the underline effect only when the link is being hovered.

You can adjust the selectors (nav a) to match your HTML structure, and modify the text-decoration property to achieve your desired style.
